riva/proto/riva_nmt.proto

```diff
@@ -137,7 +137,12 @@ message TranslateTextRequest {
   string model = 2;
   string source_language = 3;
   string target_language = 4;
-
+  //This field contains list of words or phrases which are not to be translated 
+  //and also a list of words and phrases which are to be custom translated.
+  //The words which are be custom translated are present as "<original>##<translation>"
+  //the words which are not to be translated are present as "<word>".
+  //All these words and phrases are separted by commas in dnt_phrases_string.
```

Comment:
Suggestion for the text.

// A list of words or phrases that are not to be translated or
// to be custom translated by the pipeline. Words to be custom translated
// should be specified as "<word>##<custom_translation>" and 
// words not be translated should be specified as "<word>".

Reply:
Ok